0.8.3 updates (#379)
dortania/OpenCore-Install-Guide: why-oc.md

* Does not support hiding specific drives
* Does not support changing settings within OpenCore's menu
* Does not patch PCIRoot UID value

> * Does not support macOS-only ACPI hex patching

Reviewer: I'd just drop `hex` and leave it at `macOS only ACPI patching`

* **Note**: This is not legal advice, so please make the proper assessments yourself and discuss with your lawyers if you have any concerns.

### Does macOS support NVIDIA GPUs?

> Due to issues revolving around NVIDIA support in newer versions of macOS, many users have somehow come to the conclusion that macOS never supported NVIDIA GPUs and don't at this point. Apple did still maintain and support Macs with NVIDIA GPUs like the 2013 MacBook Pro with a Kepler dGPU until they released Monterey. macOS Monterey beta 7 officially removed support for Kepler and so all NVIDIA GPUs, while there are community-made patches to bring back support, they require SIP, System Integrity Protection which is, like the name suggests, a big part of security in macOS, partially disabled which might not be ideal for everyone.

Reviewer: ```suggestion
Due to issues revolving around NVIDIA support in newer versions of macOS, many users come to the conclusion that macOS never supported NVIDIA GPUs. Apple supported Macs with NVIDIA GPUs (such as the 2013 MacBook Pro with a Kepler dGPU) until the release of Monterey Beta 7. While there are community-made patches to bring back support, they require SIP (System Integrity Protection) to be disabled, disabling important security features in macOS.
```
